src/rpms.sh

```diff
@@ -469,7 +517,7 @@ rlCheckRequirements() {
 
 This is just a bit smarted wrapper of
 
-C<rlCheckRequirements $(rlGetMakefileRequires)>
+C<rlCheckRequirements $(rlGetYAMLdeps 'requires|recommends') || rlCheckRequirements $(rlGetMakefileRequires)>
```

Comment:
"requires|recommends" should be "require|recommend"

Reply:
fixed by https://github.com/beakerlib/beakerlib/pull/75/commits/2731beeff30a9dc9a748a026ceb1409b08797c5f